### DISCUSSION TITLE
Is there any specific reason Aspire templates do not use CentralPackageManagement?

### DISCUSSION THREAD

**Discussion by En3Tho** [microsoft/aspire#1150] [Ideas]
I have a tool that does the migration but I would have expected templates to use this feature by default

**Comment by DamianEdwards** [comment]
Central Package Management is a solution level mechanism and Aspire can be added to existing solutions. Better integration of CPM with the .NET templating story is being tracked by NuGet at https://github.com/NuGet/Home/issues/12629

**Reply by En3Tho** [reply]
Thank you
